src/main/java/hwr/oop/riddler/logic/SudokuValidator.java

```diff
@@ -0,0 +1,54 @@
+package hwr.oop.riddler.logic;
+
+import hwr.oop.riddler.model.Sudoku;
+import hwr.oop.riddler.model.component.Cell;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SudokuValidator {
```

Comment:
Vielleicht noch checken ob values zwischen 1 und 9 sind?

Reply:
Das kann theoretisch aber auch in die Cell Logic, z.B. in eine setValue Methode und in den Konstruktor